examples/node/hello-sdl-tcp/config.js

```diff
@@ -0,0 +1,38 @@
+/*
+* Copyright (c) 2019, Livio, Inc.
```

Comment:
```suggestion
* Copyright (c) 2020, Livio, Inc.
```
Please change throughout!